src/Hazelcast.Net.Tests/Sql/SqlQueryResultTests.cs

```diff
@@ -173,5 +175,25 @@ await foreach (var row in result)
                 Assert.Throws<SerializationException>(() => row.GetValue<DummyPortable>());
             }
         }
+
+        [Test]
+        [ServerConditionAttribute("5.0")]
+        public async Task SqlErrorHasSuggestion()
+        {
+            string dummyMapName = "testingMap";
+            var map = await Client.GetMapAsync<int, string>(dummyMapName);
+            await map.PutAsync(0, "some value");
+
+            try
+            {
+                //query the map without creating map to get exception with suggestion in it.
```

Comment:
without creating _mapping_, right? because we created the map.

Reply:
yeah, good catch!